SchedulerJobs/SchedulerJobs.Services/CloseConferenceService.cs

```diff
@@ -25,11 +25,11 @@ public async Task<int> CloseConferencesAsync(DateTime date)
             if (conferences != null && conferences.Any())
             {
                 conferenceCount = conferences.Count;
-                conferences.ForEach(async s => {
-
+                conferences.ForEach(async s =>
+                {
                     await _videoApiService.CloseConference(s.Id);
                     await _videoApiService.RemoveVirtualCourtRoom(s.HearingRefId);
```

Comment:
why are we making a seperate call to remove a virtual courtroom?

Reply:
just because it different resources and different controller

Reply:
it shouldn't be exposed. The kinly room should be disposed of by the close conference endpoint

Reply:
Done, refactoring according to following changes in video api.